packages/http/src/forwarder/index.ts

```diff
@@ -27,14 +28,48 @@ const forward: IPrismComponents<IHttpOperation, IHttpRequest, IHttpResponse, IHt
           query: input.url.query,
         }),
         {
+          body: serializeBody(input.body),
+          method: input.method,
           headers: defaults(omit(input.headers, ['host', 'accept']), {
             accept: 'application/json, text/plain, */*',
             'user-agent': `Prism/${prismVersion}`,
           }),
         }
       );
-    }, toError),
+    }, Either.toError),
     TaskEither.chain(parseResponse)
   );
 
 export default forward;
+
+function serializeBody(body: unknown) {
+  return pipe(
+    Option.fromNullable(body),
+    Option.chain(body =>
+      pipe(
+        Option.some(body),
+        Option.chain(body =>
+          pipe(
+            body,
+            Option.fromPredicate(body => typeof body === 'object'),
+            Option.chain(body =>
+              pipe(
+                Either.stringifyJSON(body, Either.toError),
+                Option.fromEither,
+                Option.alt(() => Option.some(''))
+              )
+            )
+          )
+        ),
+        Option.alt(
+          () =>
+            pipe(
+              body,
+              Option.fromPredicate(body => typeof body === 'string')
```

Comment:
Let's use a typeguard instead of a cast

```suggestion
              Option.fromPredicate((body): body is string => typeof body === 'string')
```